unyt/dask_array.py

```diff
@@ -13,12 +13,15 @@
 from unyt._on_demand_imports import _dask as dask
 
 __doctest_requires__ = {
-    ("dask_array.unyt_from_dask"): ["dask"],
+    ("unyt_from_dask", "reduce_with_units", "unyt_dask_array.to_dask"): ["dask"],
```

Comment:
Ah, ya. This was totally wrong! I suspect I got it working early on and then forgot to update it as the PR evolved. And then missed it cause of adding dask to all the tox envs as you described. Thanks for the fix!